Tests/InvokeChanges.Tests.ps1

```
@@ -0,0 +1,351 @@
+    function ImportRegistryFile { param($Message, $RegistryFile) }
+    function RemoveApps { param($Apps) }
+    function Disable-TelemetryScheduledTasks {}
+    function Enable-TelemetryScheduledTasks {}
+    function GenerateAppsList { @() }
+    function GetFriendlyTargetUserName { 'current user' }
+    function EnableStoreSearchSuggestionsForAllUsers {}
+    function EnableStoreSearchSuggestions { param($StoreAppsDatabase) }
+    function GetStoreAppsDatabasePathForUser { param($UserName) 'store.db' }
+    function GetUserName { 'Alice' }
+    function DisableWindowsFeature { param($FeatureName) }
+    function New-RegistrySettingsBackup { param($ActionableKeys, $ExtraFeatures) }
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'New-RegistrySettingsBackup' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)